src/containers/pages/FocusInvestigation/active/tests/index.test.tsx

```diff
@@ -1,5 +1,6 @@
 import { library } from '@fortawesome/fontawesome-svg-core';
 import { faExternalLinkSquareAlt } from '@fortawesome/free-solid-svg-icons';
+import DrillDownTable from '@onaio/drill-down-table/dist/types';
```

Comment:
Could you remove all unused imports from here?

Reply:
done